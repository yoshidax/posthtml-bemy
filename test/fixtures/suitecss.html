<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <title>Document</title>
</head>

<body>
    <div class="_block
                --modifire
                --modifire2">
        <div class="-element --modifire">
            <div class="_blockChild">
                <div class="-element1">1</div>
                <div class="-element2">2</div>
                <div class="-element3">3</div>
                <div class="something">something</div>
            </div>
        </div>
    </div>
    <div class="_block">
        <div class="    _foo   --bar">
            <div class="   -bar  --barbar"></div>
        </div>
    </div>
    <div class="_block
                --modifire
                --modifire2">
        <div class="-element --modifire">
            <div class="_blockChild">
                <div class="-element1"></div>
                <div class="-element2"></div>
                <div class="-element3"></div>
                <div class="something">something</div>
            </div>
        </div>
    </div>
</body>

</html>
