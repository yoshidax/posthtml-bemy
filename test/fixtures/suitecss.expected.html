<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <title>Document</title>
</head>

<body>
    <div class="block
                block--modifire
                block--modifire2">
        <div class="block-element block-element--modifire">
            <div class="blockChild">
                <div class="blockChild-element1">1</div>
                <div class="blockChild-element2">2</div>
                <div class="blockChild-element3">3</div>
                <div class="something">something</div>
            </div>
        </div>
    </div>
    <div class="block">
        <div class="    foo   foo--bar">
            <div class="   foo-bar  foo-bar--barbar"></div>
        </div>
    </div>
    <div class="block
                block--modifire
                block--modifire2">
        <div class="block-element block-element--modifire">
            <div class="blockChild">
                <div class="blockChild-element1"></div>
                <div class="blockChild-element2"></div>
                <div class="blockChild-element3"></div>
                <div class="something">something</div>
            </div>
        </div>
    </div>
</body>

</html>
